src/react-native-shadow-stack/ShadowView.js

```diff
@@ -1,22 +1,17 @@
-import React, { useMemo } from 'react';
+import React from 'react';
 import { StyleSheet, View } from 'react-native';
-import { Shadow } from 'react-native-neomorph-shadows';
 
 const ShadowView = props => {
   if (ios || props.elevation > 0) {
     return <View {...props} />;
+  } else {
```

Comment:
super minor, but this `else` is unnecessary 

not reallyyyy necessary for you to change tho, its super minor